aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/PluginReflectionUtility.java

```diff
@@ -222,50 +223,97 @@ private static void populateRuntimeContext(Object handler, Field field) {
 
     /**
      * Retrieves annotations of a {@code Field} instance as a sequential {@code Stream}
-     * @param field The field to analyze
+     * @param member The field to analyze
      * @return Stream of annotation objects,
      */
-    public static Stream<Class<? extends Annotation>> getFieldAnnotations(Field field){
-        return Arrays.stream(field.getDeclaredAnnotations())
-                .map(Annotation::annotationType);
+    public static Stream<Class<? extends Annotation>> getMemberAnnotationClasses(Member member) {
```

Comment:
I think these methods for extracting member annotation should be a part of toolkit public API.